velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -330,6 +363,17 @@ bool SubstraitToVeloxPlanValidator::validate(
     }
   }
 
+  // Validate supported aggregate functions.
+  std::unordered_set<std::string> unsupportedFuncs = {"collect_list"};
+  for (const auto& funcSpec : funcSpecs) {
+    auto funcName = subParser_->getSubFunctionName(funcSpec);
+    if (unsupportedFuncs.find(funcName) != unsupportedFuncs.end()) {
+      std::cout << "Validation failed due to " << funcName
+                << " was not supported in WindowRel." << std::endl;
```

Comment:
Why not supported in `WindowRel `here?

Reply:
I'm not sure why it is in the black list. It is just moved from Scala black list here.

https://github.com/oap-project/gluten/pull/1360/files#diff-913561b8f988672d65b0436d9cf74cdd3c453f14af58376b0a3c0e3a2a5875abL55-L57